ebms-send-in/src/main/kotlin/no/nav/emottak/fellesformat/FellesFormatWrapper.kt

```diff
@@ -42,15 +34,15 @@ private fun createFellesFormatMottakEnhetBlokk(sendInRequest: SendInRequest): EI
         it.ebAction = sendInRequest.addressing.action
         it.ebService = sendInRequest.addressing.service
         it.ebRole = sendInRequest.addressing.from.role
-        it.avsender = "TODO"
-        it.avsenderRef = "TODO"
+        it.avsender = "TODO1" // Hentes fra from. Usikker på hvilket felt siden det kan være flere.
+        it.avsenderRef = "TODO2" // Hentet fra cert: Eksempelverdi: "SERIALNUMBER=132547698, CN=Blå &amp; Bjørnebær AS, O=Blå &amp; Bjørnebær AS, C=NO"
         it.mottaksId = sendInRequest.messageId
         it.mottattDatotid = Instant.now().toXMLGregorianCalendar()
-        it.ediLoggId = "TODO"
-        it.avsenderFnrFraDigSignatur = "TODO"
-        it.avsenderOrgNrFraDigSignatur = "TODO"
-        it.herIdentifikator = "TODO"
-        it.orgNummer = "TODO"
-        it.meldingsType = "TODO"
+        it.ediLoggId = sendInRequest.messageId
```

Comment:
Jeg er ikke sikker på hvordan mottak håndterer dette, og jeg vet ikke om dette er i det hele tatt mulig, men kan det hende at messageId er null? Hvis den kan være null i en eller annen flyt, lurer jeg på om det kan være lurt å sende en empty string i stedet for null som kommer i XML som teksten null. Jeg vet ikke da, bare lufter tanker :sweat_smile: